### ISSUE TITLE
Hookdeck Telemetry

### ISSUE THREAD

**Issue by alexluong** [hookdeck/outpost#142]
- [ ] Errors & crash report
- [ ] Usage statistics
- [ ] Opt-out

---

Questions:

- How (which tool) should we track error & crash report? Usage statistics?
- What are the key metrics to track?

**Comment by leggetter** [comment]
@alexluong - this is currently labeled as a question. However, I'm assuming there is implementation required here?

**Comment by alexluong** [comment]
@leggetter yes, sorry the "question" label is to mark things that require an answer. Since @alexbouchardd already addressed that, lemme remove the label here. It's still something that needs to be done.

**Comment by leggetter** [comment]
@alexluong disable Sentry for now. Enable with sensible default and configurability in BETA.

**Comment by alexluong** [comment]
implemented #252
